### ISSUE TITLE
make all fails

### ISSUE THREAD

**Issue by rolfschumacher** [PrzemekWirkus/sbs2xml-conv#1]
Tried to do a make all under Linux, seems the Makefile was written for cygwin.
In Makefile I commented out "# SHELL=C:/Windows/System32/cmd.exe" relying on the definition in the environment as SHELL=/bin/bash.
Now I get
rsc@rolf24:~/workspace/sbs2xml-conv$ make all
flex lexer.l
gawk "/#define YY_BUF_SIZE/{gsub(/16384/, "(1024_1024)")};{print}" lex.yy.c > lex.yy.c.new
/bin/sh: 1: Syntax error: "(" unexpected
Makefile:37: recipe for target 'flex' failed
make: *_\* [flex] Error 2
rsc@rolf24:~/workspace/sbs2xml-conv$

**Comment by petreDanielMarian** [comment]
Hey,
I had the same problem, but with the help of some great guys from stack overflow i managed to compile the file and make an executable which generates the .xml

The StackOverflow thread which contains some more info: https://stackoverflow.com/questions/51206691/error-on-unexpected-bracket-in-gawk/51210370#51210370

**Comment by PrzemekWirkus** [comment]
@rolfschumacher Indeed, Makefile is written for Windows/Cygwin users. At the time it was my development platform so Linux version was not provided.

**Comment by xena7201** [comment]
Hi! I tried to parse an .sbs file and I'd get "syntax error". 

I am using Rhapsody 8.2.1 and I think the issue is somehow related to a possible syntax difference between the repository files generated by the two Rhapsody versions, 7.5 and 8.2.1.

Could you help me find a solution for this? 
[ComponentX.zip](https://github.com/PrzemekWirkus/sbs2xml-conv/files/7686457/ComponentX.zip)

---
I leave a sample file as attachment.
